test/js/pageobjects/Welcome.js

```diff
@@ -2,6 +2,13 @@ import Base from './BasePage.js'
 
 class WelcomeScreen extends Base{
     getPrimaryBtn() { return this.$('.onfido-sdk-ui-Button-button'); }
+    get welcomeTitle() { return this.$('.onfido-sdk-ui-Title-titleSpan')}
+    get welcomeSubtitle() { return this.$('.onfido-sdk-ui-Welcome-text')}
+    get footer() { return this.$('.onfido-sdk-ui-Theme-footer')}
+    get primaryBtn() { return this.$('.onfido-sdk-ui-Button-button')}
+
+    copy = (lang="en") =>
```

Comment:
I was going to ask why do we need to `copy` (i assumed it was because we were mutating the instance) but it seems my mistake was probably the name.

Are you sure you mean `copy`? Isn't this more like a fetcher? Because in fact we aren't copying right? We are always using the same instance?

Reply:
I'm improving that in https://github.com/onfido/onfido-sdk-ui/pull/635/commits/d13a76a4463c4de5ca8d0a8e19975f8973f6fa6b